Library/Formula/openwsman.rb

```diff
@@ -18,6 +18,8 @@ class Openwsman < Formula
   depends_on "pkg-config" => :build
   depends_on "openssl"
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!